packages/flame/lib/src/game/mixins/game.dart

```diff
@@ -110,16 +110,29 @@ mixin Game on Loadable implements Projector {
       ''');
     }
     _gameRenderBox = gameRenderBox;
+    onAttach();
   }
 
+  /// Called when the game has been attached this can be overriden
+  /// to add logic that requires that the game to be already attached
+  /// to the widget tree
```

Comment:
```suggestion
  /// Called when the game has been attached. This can be overriden
  /// to add logic that requires the game to already be attached
  /// to the widget tree.
```